GeneratorConfig/PokemonSwSh/GodEggDuplication.json

```diff
@@ -1,25 +0,0 @@
-﻿{

```

Comment:
Don't delete this file. The program does actually exist, but I think it's hidden.

Reply:
Added back